README.md

```diff
@@ -21,6 +21,21 @@ registry.example.com/nginx/nginx:latest
 ```
 
 ## NOTICE
+Kubernetes APIs upgrade
+
+>**Deprecated APIs in v1.22** https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22 Because the end of support for some beta APIs the code received some important updates. Now you need to specify a signer for your Certificate Signing Request. You can take a look on the official documentation about API changes https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22.
```

Comment:
Need to use markdown hyperlink syntax for links

Reply:
done!